### PR TITLE
changed init players to people on hall

### PR DESCRIPTION
**Pull request by anpere**

### CODE REVIEW

**Comment by seveneightn9ne** [comment]
:-1:

File: scripts/game-state.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ module.exports = GameState
 function GameState(dispatcher) {
     Store.mixin(this)
 
-    this.playerNames = ['Miles', 'Jess', 'Brandon', 'Ciara', 'Chris']
+    this.playerNames = ['Miles', 'Jess', 'Andres', 'Carolyn', 'Druck','Akshat']
```
Comment (seveneightn9ne) [inline scripts/game-state.js]: pls fix

Comment (mlsteele) [inline scripts/game-state.js]: missing space
:-1: 

Also Drück needs an ümlaüt.

**Comment by seveneightn9ne** [comment]
do the umlaut homie

**Comment by anpere** [comment]
:))

**Comment by mlsteele** [comment]
:+1: fantastic

**Pull request merged**